2-mandatory/7-eligible-students.js

```diff
@@ -14,7 +14,15 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
+  let eligibleStudentsArr = []
+  for (let attendance of attendances) {
+    if (attendance.attendance >= 8) {
+      eligibleStudentsArr.push(attendance.name)
+    }
+  }
+  return eligibleStudentsArr
```

Comment:
This works, and while there is not issue, can you work out a simpler way of doing this? Hint: we want to _filter_ using the condition attendance>=8